### PR TITLE
Update readme for cookbook

### PR DESCRIPTION
**Pull request by geek-yang**
Changes made to readme:

- Resize and relocate logo
- Add section about more usecases in cookbook
- Add acknowledgement with grant number

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,6 +135,8 @@ have a look at the [contribution guidelines](docs/CONTRIBUTING.md).
 
 Please use the Zenodo DOI to cite this package if you used it in your research.
 
-## Credits
+## Acknowledgements
+
+This package was developed by the Netherlands eScience Center and Vrije Universiteit Amsterdam. It was supported by the Netherlands eScience Center under grant number NLESC.OEC.2021.005.
```
Comment (BSchilperoort) [inline README.md:140]:
```suggestion
This package was developed by the Netherlands eScience Center and Vrije Universiteit Amsterdam. Development was supported by the Netherlands eScience Center under grant number NLESC.OEC.2021.005.
```

File: pyproject.toml [hunk 2]
```diff
@@ -155,6 +155,7 @@ target-version = "py38"
 
 [tool.ruff.per-file-ignores]
 "tests/**" = ["D"]
+"tests/test_preprocess.py" = ["B018"]
```
Comment (BSchilperoort) [inline pyproject.toml:158]: It's probably better to suppress this error in the specific file.

You can just add a line like the following after the imports:
`# ruff: noqa: B018`

And add a comment there to explain why it is being suppressed.

Alternatively, if this error messes with general test code, we can just add this to the previous line
```toml
"tests/**" = ["D", "B018"]
```

**Review by BSchilperoort — APPROVED** [review]
Looks good. Just some small comments :)

File: pyproject.toml (hunk 2 above)
Comment (geek-yang) [reply pyproject.toml]: Sounds good. I simply add `# ruff: noqa: B018` in that file and add some comments.

**Pull request merged**